clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -720,25 +733,17 @@ private KafkaConsumer(ConsumerConfig config,
 
             NetworkClient netClient = new NetworkClient(
                     new Selector(config.getLong(ConsumerConfig.CONNECTIONS_MAX_IDLE_MS_CONFIG), metrics, time, metricGrpPrefix, channelBuilder, logContext),
-                    this.metadata,
-                    clientId,
+                    this.metadata, clientId,
                     100, // a fixed large enough value will suffice for max in-flight requests
                     config.getLong(ConsumerConfig.RECONNECT_BACKOFF_MS_CONFIG),
                     config.getLong(ConsumerConfig.RECONNECT_BACKOFF_MAX_MS_CONFIG),
                     config.getInt(ConsumerConfig.SEND_BUFFER_CONFIG),
                     config.getInt(ConsumerConfig.RECEIVE_BUFFER_CONFIG),
                     config.getInt(ConsumerConfig.REQUEST_TIMEOUT_MS_CONFIG),
-                    time,
-                    true,
-                    new ApiVersions(),
-                    throttleTimeSensor,
+                    time, true, new ApiVersions(), throttleTimeSensor,
```

Comment:
It's better to keep the parameters aligned (having same indentation)

Reply:
NPath Complexity is a boon. Checkstyle believes KafkaConsumer's constructor contains too many lines. I did this to make the method shorter.